2021-10-21 00:08:45,605 INFO    MainThread:66205 [wandb_setup.py:_flush():69] setting env: {'project': 'exercise_14', 'run_group': 'dev'}
2021-10-21 00:08:45,605 WARNING MainThread:66205 [wandb_setup.py:_flush():69] could not save program above cwd: /Users/obara/miniforge3/envs/mlflow-827614215fe534cb79b9b720eb621b7f7070f8a8/bin/pytest
2021-10-21 00:08:45,605 INFO    MainThread:66205 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-21 00:08:45,605 INFO    MainThread:66205 [wandb_init.py:_log_setup():342] Logging user logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/check_data/wandb/run-20211021_000845-2mit821n/logs/debug.log
2021-10-21 00:08:45,606 INFO    MainThread:66205 [wandb_init.py:_log_setup():343] Logging internal logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/check_data/wandb/run-20211021_000845-2mit821n/logs/debug-internal.log
2021-10-21 00:08:45,606 INFO    MainThread:66205 [wandb_init.py:init():375] calling init triggers
2021-10-21 00:08:45,606 INFO    MainThread:66205 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-21 00:08:45,606 INFO    MainThread:66205 [wandb_init.py:init():424] starting backend
2021-10-21 00:08:45,606 INFO    MainThread:66205 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-10-21 00:08:45,621 INFO    MainThread:66205 [backend.py:ensure_launched():135] starting backend process...
2021-10-21 00:08:45,627 INFO    MainThread:66205 [backend.py:ensure_launched():139] started backend process with pid: 66252
2021-10-21 00:08:45,629 INFO    MainThread:66205 [wandb_init.py:init():429] backend started and connected
2021-10-21 00:08:45,641 INFO    MainThread:66205 [wandb_init.py:init():477] updated telemetry
2021-10-21 00:08:45,642 INFO    MainThread:66205 [wandb_init.py:init():500] communicating current version
2021-10-21 00:08:46,071 INFO    MainThread:66205 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-21 00:08:46,071 INFO    MainThread:66205 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-21 00:08:46,302 INFO    MainThread:66205 [wandb_init.py:init():540] starting run threads in backend
2021-10-21 00:08:46,759 INFO    MainThread:66205 [wandb_run.py:_console_start():1601] atexit reg
2021-10-21 00:08:46,760 INFO    MainThread:66205 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-10-21 00:08:46,760 INFO    MainThread:66205 [wandb_run.py:_redirect():1480] Redirecting console.
2021-10-21 00:08:46,760 INFO    MainThread:66205 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-21 00:08:46,760 INFO    MainThread:66205 [wandb_init.py:init():565] run started, returning control to user process
2021-10-21 00:08:48,950 INFO    MainThread:66205 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-10-21 00:08:48,950 INFO    MainThread:66205 [wandb_run.py:_restore():1543] restore
2021-10-21 00:08:51,809 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 1047
}

2021-10-21 00:08:52,851 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 1047
}

2021-10-21 00:08:53,068 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 1047
}

2021-10-21 00:08:53,342 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 7067
}

2021-10-21 00:08:53,447 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 7067
}

2021-10-21 00:08:53,576 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 7067
}

2021-10-21 00:08:53,683 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:08:53,789 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:08:53,896 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:08:53,999 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:08:54,105 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:08:54,212 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:08:54,369 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:08:54,475 INFO    MainThread:66205 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:08:55,557 INFO    MainThread:66205 [wandb_run.py:_show_files():1915] logging synced files
